pithos/pandora/pandora.py

```diff
@@ -586,3 +586,4 @@ def __init__(self, resultType, d):
         elif resultType == 'genre':
             self.stationName = d['stationName']
 
+
```

Comment:
__[flake8]__
* [W391] blank line at end of file

<!-- Comment by [SideCI](https://sideci.com) -->